mongothon/document.py

```diff
@@ -204,8 +204,8 @@ def update(self, other=None, **kwargs):
                 self[key] = other[key]
 
         if kwargs:
-            for key, in kwargs:
-                self[key] = kwargs[key]
```

Comment:
It didn't :trollface:, but it's pretty rare to see keyword args passed to `dict.update` in the wild...